libhoney/client.py

```diff
@@ -128,6 +153,7 @@ def send(self, event):
             raise SendError(
```

Comment:
does this deserve a `self.log` too?

Reply:
I'd love to stop raising an exception entirely and just log.